src/test/scala/uk/gov/nationalarchives/tdr/api/routes/FileRouteSpec.scala

```diff
@@ -50,6 +50,24 @@ class FileRouteSpec extends TestContainerUtils with Matchers with TestRequest {
       expectedResponse.data.get.addFilesAndMetadata should equal(response.data.get.addFilesAndMetadata)
   }
 
+  "The api" should "add files and metadata entries for files and directories" in withContainers {
+    case container: PostgreSQLContainer =>
+      val consignmentId = UUID.fromString("f1a9269d-157b-402c-98d8-1633393634c5")
+      val utils = TestUtils(container.database)
+      (clientSideProperties ++ staticMetadataProperties.map(_.name)).foreach(utils.addFileProperty)
+      utils.createConsignment(consignmentId, userId)
+
+      val response = runTestMutationFileMetadata("mutation_alldata", validUserToken())
```

Comment:
The val `response` is no longer used, as switched to checking with counts? Was there a reason for no longer using as JSON response to check the expected result?

Reply:
I've taken it out. The test above it checks the response but the response is basic, just `{ "fileId": "...", "matchId": 1 }` so I added in this test to check that all the database updates are being done properly.